meta-iot2000-example/recipes-example/iot2000setup/iot2000setup_1.0.bb

```diff
@@ -1,23 +1,27 @@
 SUMMARY = "Copy iot2000setup files to file system"
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://${IOT2000_MIT_LICENSE};md5=838c366f69b72c5df05c96dff79b35f2"
+inherit update-rc.d 
 
 SRC_URI = "file://expandfs.sh file://iot2000setup.py"
 
 FILES_${PN} += " \
-	${sysconfdir}/iot2000setup/expandfs.sh \
 	${sysconfdir}/iot2000setup/iot2000setup.py \
```

Comment:
Hm, I'm still not sure if putting this under `/etc/<whatever>` is the right place. All those files could as well be placed so that they better fit the [FHS](http://www.pathname.com/fhs/pub/fhs-2.3.html), can't they?